examples/demos/components/Editors/DropDownEditor.tsx

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import React, { forwardRef, useCallback, useImperativeHandle, useState } from 'react';
+import { Column, Editor } from '../../../../src';
+
+interface Option {
+  id: string;
+  title: string;
+  value: string;
+  text: string;
+}
+
+interface DropDownEditorProps<TRow> {
+  column: Column<TRow>;
+  options: Array<Option | string>;
+  value?: string;
+  onBlur(): void;
+}
+
+type DropDownEditorHandle = Pick<Editor<{[key: string]: string | undefined}>, 'getInputNode' | 'getValue'>;
+
+function DropDownEditorWithRef<TRow>({ column, value, onBlur, options }: DropDownEditorProps<TRow>, ref: React.Ref<DropDownEditorHandle>): JSX.Element {
+  const selectRef = React.createRef<HTMLSelectElement>();
+  const [optionsHeight, setOptionsHeight] = useState<number | undefined>();
+
+  const onFocus = useCallback(() => {
+    if (selectRef && selectRef.current) {
+      selectRef.current.size = options.length;
```

Comment:
What does this do?

Reply:
by default, when you open this `<select />` editor, it opens, but acts like not opened. Because the `<select />` tag won't open the `options` inside, so it makes the users feeling it's broken.